src/dropdown.test.visuals.ts

```diff
@@ -314,51 +414,6 @@ for (const story of stories.Dropdown) {
             );
           });
         });
-
-        test.describe('<glide-core-dropdown-option>.editable', () => {
```

Comment:
Moved to `test.describe('size="small"')` because `size` is what causes the greatest visual change to `editable`.